silverstripe/silverstripe-linkfield
# FIX Update LinkField if Elemental version status was updated

## Description

This fix updates the component if the elemental state has been updated. 

## Manual testing steps


- I create a new Elemental block with LinkField;
- I create a new Link;
- I see "Draft" badge in LinkField;
- I publish the block (not a Page);
- I still see "Draft" badge in LinkField;
- I refresh the Page and then a badge disappears.

## Issues

- https://github.com/silverstripe/silverstripe-linkfield/issues/256

## Pull request checklist

    - See [picking the right version](https://docs.silverstripe.org/en/contributing/code/#picking-the-right-version)
    - Small amounts of additional linting are usually okay, but if it makes it hard to concentrate on the relevant changes, ask for the unrelated changes to be reverted, and submitted as a separate PR.
